clients/netcore/APIAuthHandler.cs

```diff
@@ -82,13 +82,13 @@ public APIAuthHandler(string authUrl, string scope, string clientId, string clie
 			request.Content = postBody;
 			var byteArray = new UTF8Encoding().GetBytes($"{_clientId}:{_clientSecret}");
 			request.Headers.Authorization = new AuthenticationHeaderValue("Basic", Convert.ToBase64String(byteArray));
-			var response = await base.SendAsync(request, cancellationToken);
+			var response = await base.SendAsync(request, cancellationToken).ConfigureAwait(false);
 
 			if (response.StatusCode != HttpStatusCode.OK) {
 				throw new Exception("Tributech API Authorization failed");
 			}
 
-			var responseBodyString = await response.Content.ReadAsStringAsync();
+			var responseBodyString = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
```

Comment:
```suggestion
			var responseBodyString = await response.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
```

Add cancellation token.

Reply:
only available in .net5 not in netstandard2.0
https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpcontent.readasstringasync?view=netstandard-2.0